### PR TITLE
doc: updated the reference to reflect the correct build cmds.

### PR DESCRIPTION
**Pull request by AdithyaBaglody**
In 1.9 client-sdk moved from make build system to cmake.
This patch reflects the same in the reference doc.

Signed-off-by: Adithya Baglody <adithya.nagaraj.baglody@intel.com>

### CODE REVIEW

**Review by trbehera — APPROVED** [review]

File: docs/client-sdk/client-sdk-reference-guide.md [hunk 1]
```diff
@@ -185,20 +184,14 @@ Table 1.	Prerequisites
...
 |    Software                                      |    ·      Linux\* Ubuntu\* 16.04 / Ubuntu\* 18.04 using   OpenSSL\* 1.1.1f <br/>·      JDK11  <br/> ·      SDO release binaries package (for running test servers) <br/>     o The root directory of extracted binary package is referred to as <sdo_sdk_binaries> in subsequent sections. <br/>   ·      SDO Client SDK alpha-release binary package <br/>   o The root directory of extracted binary package is referred to as < SDOClientSDK> in subsequent sections. <br/>  ·      SDO Client SDK alpha-release sources package <br/>  o The root directory of extracted source code is referred to as < SDOClientSDK> in subsequent sections. <br/>  ·      Apache Maven\* (for building sample Java\* based test owner   service)   <br/>    Additional Ubuntu\* 16.04/18.04  prerequisites can be found in the   <releases source>/SDOClientSDK/README.md     |
```
Comment (KiranSukhavasi) [inline docs/client-sdk/client-sdk-reference-guide.md:186]: remove   \* Ubuntu\* 16.04. this is in 2 places in this parah.

**Review by v-vasavi — COMMENTED** [review]
On line 194- do we still need this - based on the device attestation method being used on the field.
since it is only ECDSA now?

**Review by KiranSukhavasi — CHANGES_REQUESTED** [review]

File: docs/client-sdk/client-sdk-reference-guide.md (hunk 1 above)
Comment (trbehera) [inline docs/client-sdk/client-sdk-reference-guide.md:186]: OpenSSL version 1.1.1g

**Review by trbehera — CHANGES_REQUESTED** [review]

File: docs/client-sdk/client-sdk-reference-guide.md [hunk 2]
```diff
@@ -185,20 +184,14 @@ Table 1.	Prerequisites
...
 The following subsections provide the steps to run the binary package.
 
-#### Get the Device Private Key (EPID or ECDSA P-256 based)
-The SDK requires a device Private Key as input for device attestation process (to prove itself to Rendezvous or Owner Server during TO1 or TO2 protocol). The key could be based on EPID or ECDSA (on curve P-256/P-384) based on the device attestation method being used on the field. This key must be stored in a specific file and is read by the SDK on startup. 
-
-_**For EPID based device-attestation method:**_   
-
-In case of EPID, place the EPID private Key with the name epidprivkey.dat in the following location:
-`data/epidprivkey.dat`
+#### Get the Device Private Key (ECDSA P-256 based)
```
Comment (trbehera) [inline docs/client-sdk/client-sdk-reference-guide.md:193]: (ECDSA P-256 based) ==> (ECDSA based)

**Comment by trbehera** [comment]
> On line 194- do we still need this - based on the device attestation method being used on the field.
> since it is only ECDSA now?

This would be required, as we still need to create device keys for either ECDSA P-256 or ECDSA P-384.

File: docs/client-sdk/client-sdk-reference-guide.md (hunk 1 above)
Comment (nbmaiti) [reply docs/client-sdk/client-sdk-reference-guide.md:186]: this is taken care in another patch PR39

File: docs/client-sdk/client-sdk-reference-guide.md (hunk 2 above)
Comment (nbmaiti) [reply docs/client-sdk/client-sdk-reference-guide.md:193]: Should be in next set of patch PR39

File: docs/client-sdk/client-sdk-reference-guide.md (hunk 1 above)
Comment (nbmaiti) [reply docs/client-sdk/client-sdk-reference-guide.md:186]: this is taken care in another patch PR39. This patch is related to EPID removal only

**Review by KiranSukhavasi — APPROVED** [review]

**Review by trbehera — APPROVED** [review]

**Pull request merged**